lnbits/core/views/payment_api.py

```diff
@@ -145,60 +143,42 @@ async def api_payments_create_invoice(data: CreateInvoice, wallet: Wallet):
         # do not save memo if description_hash or unhashed_description is set
         memo = ""
 
-    async with db.connect() as conn:
-        payment_hash, payment_request = await create_invoice(
-            wallet_id=wallet.id,
-            amount=data.amount,
-            memo=memo,
-            currency=data.unit,
-            description_hash=description_hash,
-            unhashed_description=unhashed_description,
-            expiry=data.expiry,
-            extra=data.extra,
-            webhook=data.webhook,
-            internal=data.internal,
-            conn=conn,
-        )
-        # NOTE: we get the checking_id with a seperate query because create_invoice
-        # does not return it and it would be a big hustle to change its return type
-        # (used across extensions)
-        payment_db = await get_standalone_payment(payment_hash, conn=conn)
-        assert payment_db is not None, "payment not found"
-        checking_id = payment_db.checking_id
-
-    invoice = bolt11.decode(payment_request)
+    payment = await create_invoice(
+        wallet_id=wallet.id,
+        amount=data.amount,
+        memo=memo,
+        currency=data.unit,
+        description_hash=description_hash,
+        unhashed_description=unhashed_description,
+        expiry=data.expiry,
+        extra=data.extra,
+        webhook=data.webhook,
+        internal=data.internal,
+    )
 
-    lnurl_response: Union[None, bool, str] = None
+    # lnurl_response is not saved in the database
     if data.lnurl_callback:
         headers = {"User-Agent": settings.user_agent}
         async with httpx.AsyncClient(headers=headers) as client:
             try:
                 r = await client.get(
                     data.lnurl_callback,
-                    params={
-                        "pr": payment_request,
-                    },
+                    params={"pr": payment.bolt11},
                     timeout=10,
                 )
                 if r.is_error:
-                    lnurl_response = r.text
+                    payment.extra["lnurl_response"] = r.text
                 else:
                     resp = json.loads(r.text)
                     if resp["status"] != "OK":
-                        lnurl_response = resp["reason"]
+                        payment.extra["lnurl_response"] = resp["reason"]
                     else:
-                        lnurl_response = True
+                        payment.extra["lnurl_response"] = True
             except (httpx.ConnectError, httpx.RequestError) as ex:
                 logger.error(ex)
-                lnurl_response = False
+                payment.extra["lnurl_response"] = False
 
-    return {
-        "payment_hash": invoice.payment_hash,
-        "payment_request": payment_request,
-        "lnurl_response": lnurl_response,
```

Comment:
- `lnurl_response` is not in extra which breaks backwards compability

Reply:
add to breaking change readme